app/models/event_blacklisting.rb

```diff
@@ -0,0 +1,76 @@
+class EventBlacklisting < ActiveRecord::Base
+  default_value_for :enabled, true
+  after_save        :queue_sync_filtered_events
+  after_destroy     :queue_sync_filtered_events
+  after_destroy { |r| $audit_log.success("Event filter [#{r.event_name}] for provider #{r.provider_model} with ID [#{r.ems_id}] has been deleted by user [#{self.class.current_userid}]") }
+
+  YAML_FILE ||= Rails.root.join("db", "fixtures", "#{self.to_s.pluralize.underscore}.yml")
```

Comment:
You do not need 'self' here and the to_s should be name: `name.pluralize.underscore`